### PR TITLE
Fix allocate additional buffer for encoding stompFrame without readab…

### PR DESCRIPTION
**Pull request by amizurov**
Motivation:

Not always STOMP frames contain any payload some times it just headers. So we wan't allocate additional buffer with NULL content for this situation.

Modification:

Modify StompSubframeEncoder to check if content is readable or not. If content is not readable just add NULL byte to encoded header buffer.

### CODE REVIEW

File: codec-stomp/src/main/java/io/netty/handler/codec/stomp/StompSubframeEncoder.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.ByteBufUtil;
+import io.netty.buffer.Unpooled;
```
Comment (normanmaurer) [inline codec-stomp/src/main/java/io/netty/handler/codec/stomp/StompSubframeEncoder.java]: @amizurov whys this import ?

Comment (amizurov) [reply codec-stomp/src/main/java/io/netty/handler/codec/stomp/StompSubframeEncoder.java]: Thx, fixed.

**Comment by normanmaurer** [comment]
@netty-bot test this please

**Comment by normanmaurer** [comment]
@netty-bot test this please

**Comment by normanmaurer** [comment]
@amizurov thanks a lot!

**Pull request merged**